Index: git-quilt/main.c
===================================================================
--- git-quilt.orig/main.c	2015-11-02 12:52:22.421359098 +0530
+++ git-quilt/main.c	2015-11-02 12:52:22.433359099 +0530
@@ -1,13 +1,16 @@
 #include<stdio.h>
+#include"main.h"
 int main()
 {
 
-	int a,b,s;
+	int a,b,s,p;
 	a=10;
 	b=20;
 	s=sum(a,b);
 	printf("Sum=%d",sum);
-	return 0;
+	p=prod(a,b);
+        printf("Prod=%d",p);
+        return 0;
 }
 
 int sum(int a,int b)
@@ -16,3 +19,10 @@
 
 	return a+b;
 }
+int prod(int a,int b)
+{
+
+
+        return a*b;
+}
+
Index: git-quilt/main.h
===================================================================
--- git-quilt.orig/main.h	2015-11-02 12:51:52.393358138 +0530
+++ git-quilt/main.h	2015-11-02 12:52:22.433359099 +0530
@@ -1 +1,2 @@
 int sum(int,int);
+int prod(int,int);
